### DISCUSSION TITLE
Daily 25/09

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#101] [Daily]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by BraianTroncoso** [comment]
Estuve refactorizando partes del codigo y ayudando a resolver bugs junto con nati. Tambien estuve estudiando android para poder terminar mis tareas de las cuales ya estoy trabajando en ellas.

**Comment by NataliaAlvarezIspc** [comment]
El fin de semana estuve con la activity de login con bastantes invonvenientes con la computadora y dificultades al momento de emular. Al tener poco conocimiento y no los mejores elementos para trabajar me lleva mas tiempo. El sábado tuvimos nuestra weekly y nos asignamos las tareas que quedaban pendientes.
Hoy segui trabajando en el pagina de login y ya la pude subir al repositorio.
